src/pages/Login.tsx

```diff
@@ -0,0 +1,225 @@
+import React from 'react';
+import Button from '@mui/material/Button';
+import CssBaseline from '@mui/material/CssBaseline';
+import TextField from '@mui/material/TextField';
+import Paper from '@mui/material/Paper';
+import Box from '@mui/material/Box';
+import Grid from '@mui/material/Grid';
+import Typography from '@mui/material/Typography';
+import { createTheme, ThemeProvider } from '@mui/material/styles';
+import login_image from '../assets/login-image.webp'
+import intercars_logo from '../assets/ic_new_logo.jpg'
+import { useForm, Controller, SubmitHandler } from "react-hook-form"
+import { IconButton } from '@mui/material';
+import InputAdornment from '@mui/material/InputAdornment';
+import Visibility from '@mui/icons-material/Visibility';
+import VisibilityOff from '@mui/icons-material/VisibilityOff';
+
+import { yupResolver } from "@hookform/resolvers/yup"
+import * as yup from "yup"
+
+const schema = yup
+    .object({
+        email: yup.string().required('Полето за имейл e задължително').email('Невалиден имейл адрес'),
+        password: yup.string().required('Полето за парола e задължително').min(4, 'Паролата трябва да e поне 4 символа').max(10, 'Паролата трябва до 10 символа')
+            .matches(/[0-9]/, 'Паролата трябва да съдържа поне една цифра')
+            .matches(/[!@#$%^&*]/, 'Паролата трябва да съдържа поне един специален знак')
+    })
+    .required()
+
+interface LoginSchema extends yup.InferType<typeof schema> {
+    email: string;
+    password: string;
+}
+
+const theme = createTheme({
+    palette: {
+        primary: {
+            // червено
+            main: '#C9022D',
+            light: '#C9022D80',
+            dark: '#88151b',
+        },
+        // черно
+        secondary: {
+            main: '#000000'
+        },
+    },
+});
+
+export default function Login() {
+    const [showPassword, setShowPassword] = React.useState(false);
+    const { control, handleSubmit, formState: { errors } } = useForm<LoginSchema>({
+        resolver: yupResolver(schema),
+    })
+
+    const onSubmit: SubmitHandler<LoginSchema> = (data) => console.log(data)
+
+    const handleClickShowPassword = () => setShowPassword((show) => !show)
+
+    const handleMouseDownPassword = (event: React.MouseEvent<HTMLButtonElement>) => {
+        event.preventDefault()
+    }
+
+    return (
+        <ThemeProvider theme={theme}>
+            <Grid container component="main" sx={{ height: '100vh' }}>
+                <CssBaseline />
```

Comment:
no need to use this, its already imported inside `App.tsx` so its global

Reply:
fixed